.gitignore

```diff
@@ -15,8 +15,8 @@
 [Ll]og/
 [Ll]ogs/
 
-# Generated file from .ttinclude
-**/Generated/*.g.cs
+# T4 Template Generated files
+!**/Generated/*.g.cs
```

Comment:
I'm confused, if they were ignored before then why do they show up normally in the repo?
Eg. here: https://github.com/CommunityToolkit/dotnet/tree/main/CommunityToolkit.Diagnostics/Generated.

Reply:
If you have ignored files already committed, only subsequent changes are ignored and the files would still be present in source control.